tgui/src/interfaces/genpop_timer.ract

```diff
@@ -0,0 +1,17 @@
+<ui-display title='Genpop Timer' button>

```

Comment:
Let me know when you're mostly done with this PR, this is a tgui 1.0 interface, I'll help you update it.